### ISSUE TITLE
Japanese handwriting recognition is not possible

### ISSUE THREAD

**Issue by yyamagit** [ddvk/remarkable-hacks#457]
Japanese handwriting recognition is not possible, because Japanese is missing from the language options for the conversion source.

**Comment by yyamagit** [comment]
Further information.
Japanese language has been shown as an option in the following settings screen.
 Language settings - Handwriting conversion language

However, in the latest version, Japanese is no longer shown as an option.
I am using rm1, and versions before 2.14.1.866 had any problems.

**Comment by ai212983** [comment]
@yyamagit Are you sure its a remarkable-hacks problem? Have you tried Japanese handwriting recognition on stock firmware?

**Comment by ddvk** [comment]
yep, a bug, there were some extra languages and with the last version things got mixed up

**Comment by yyamagit** [comment]
Thanks for confirming the issue. 
I'm not in a hurry to fix it, so I hope you can fix it when it's appropriate.

**Comment by Pazukulele** [comment]
Japanese is still missing from the language settings lists with patch 38.2.01-0. Please try to reinstall it, I really need it. Thanks!

**Comment by ddvk** [comment]
Yep, I have to fix it, something changed underneath and I haven''t figured it out yet
